src/msteams-app.css

```diff
@@ -0,0 +1,163 @@
+@charset "UTF-8";
+
+body {
+  font-family:  "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+  font-size: 1.4rem;
+  min-width: auto;
+}
+a {
+  color: #5558af;
+}
+a:hover, a:focus {
+  text-decoration: underline;
+  outline: none;
+}
+
+/* Form controls */
+.form-field-title:first-of-type {
+  margin-top: 0;
+}
+.form-field-title {
+  display: flex;
+  margin: 1rem 0 .3rem;
+}
+.form-control:focus {
+  border-bottom-color: #5558af;
+  box-shadow: none;
+  outline: 0;
+}
+.form-control {

```

Comment:
Actually I was halfway through doing something like this but it makes the css a lot more unwieldy. 

Plus, .form-control is CSS that's specific to Teams. The only stock CSS that we style is <a> and <input> and <textarea> and this is about giving the config experience a look and feel that matches Teams.